api/src/main/resources/liquibase.xml

```diff
@@ -289,6 +289,22 @@
         </sql>
     </changeSet>
 
+    <changeSet id="create-appointment-status-change-task-201709121099999" author="Anubhav">
```

Comment:
Author should be `Bahmni`. We stopped putting dev names long time back.

Reply:
Ohk was not aware of that will change to BAHMNI (just saw previous authors name :) so added